Feature/updates (#74)
infinum/eightshift-docs: website/docs/basics/browser-sync.md

```bash
[Browsersync] Proxying: https://local-url.test
[Browsersync] Access URLs:
--------------------------------------
Local: https://localhost:3000
External: https://192.168.0.25:3000
--------------------------------------
UI: http://localhost:3001
UI External: http://localhost:3001
--------------------------------------
```

Using the IP address, you can open it on any device (mobile phone or a tablet) that is connected to the same network, and you'll see how your site looks like on them.

> ## Using SSL for local development

Reviewer: I think TLS is used nowadays 🤔

> By default BrowserSync is not working with the https urls but if you are like us and you use HTTPS in your local enviroment you can simply provide additional key to the Webpack config to make it work.

Reviewer 2: ```suggestion
By default BrowserSync does not work with https urls but if you use HTTPS in your local environment you can simply provide additional key to the Webpack config to make it work.
```
